src/app-layer-ftp.c

```diff
@@ -470,9 +596,27 @@ static int FTPParseRequest(Flow *f, void *ftp_state,
 
     int direction = STREAM_TOSERVER;
     while (FTPGetLine(state) >= 0) {
-        FTPParseRequestCommand(state,
-                               state->current_line, state->current_line_len);
+
+        FTPParseRequestCommand(state, state->current_line, state->current_line_len);
+        if (state->command == FTP_COMMAND_UNKNOWN) {
```

Comment:
Does this mean we only create txs for selected commands? I'm thinking that we can recognize all commands, even if we don't know them. Then we could just track and log the unknown ones as well.

Reply:
I'll modify things so that all currently defined commands in the `enum FtpRequestCommand` are recognized.